### ISSUE TITLE
Review Timezone / UTC

### ISSUE THREAD

**Issue by klizhentas** [gravitational/teleport#2800]
## Description

Currently, the UI in teleport says that session times are displayed in UTC, however uses local time instead.

In addition to that, date picker picks the time using local time and converts to UTC. Instead, I suggest to change it to time picker instead and always use UTC as well.

**Comment by klizhentas** [comment]
Seems like everyone is in agreement then, @alex-kovoy this one goes to you then.

**Comment by benarent** [comment]
We discussed this yesterday, let's show the current UTC time in the UI. 

Example of how my local PST is off from UTC. 
![image](https://user-images.githubusercontent.com/559288/62162798-ddc18d80-b2cd-11e9-8cb2-f0f290141533.png)

**Comment by alex-kovoy** [comment]
The issue with the "stored sessions" will disappear after 4.2 release because we no longer need to have session `end` event to display it. That was the main reason for people complains because  `start` and `end` events might not fall into the same selected range. Also, due to limitation on the maximum number of returned events, some "stored sessions" might not be shown as well although both would belong to selected date range.

In other words, we are replacing `stored session` abstraction with an `Audit Log` which will display events as is.

As for DatePicker -> we will continue showing local dates inside the picker.
As for DataTable -> we will be shown UTC dates in columns.

**Comment by klizhentas** [comment]
@alex-kovoy this really does not strike me as intuitive - if we are displaying times in UTC, why do we pick times with a local date picker?

**Comment by klizhentas** [comment]
cc @benarent @alexwolfe

**Comment by benarent** [comment]
To help discuss the edge cases and visually show the issues / bugs, I've created this docs. https://docs.google.com/document/d/1dE--oGaAcU38fnM5h1DeUQjqS5e-4QRkEIclgiATfsA/edit#

**Comment by zmb3** [comment]
We display the audit log in UTC now.
